pkg/auth/authenticator/request/x509request/x509.go

```diff
@@ -60,7 +61,35 @@ func (a *Authenticator) AuthenticateRequest(req *http.Request) (user.Info, bool,
 			}
 		}
 	}
-	return nil, false, errors.NewAggregate(errlist)
+	return nil, false, kerrors.NewAggregate(errlist)
+}
+
+// Verifier implements request.Authenticator by extracting user info from verified client certificates
```

Comment:
Clarify. I think this is wrapping a RequestAuthenticator inside of another RequestAuthenticator that requires a valid client cert as opposed to pulling the user info from a client cert.